src/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -178,7 +178,7 @@ public VariantContextBuilder alleles(final String ... alleleStrings) {
     }
 
     /**
-     * Tells this builder to use this map of attributes alleles for the resulting <code>VariantContext</code>
+     * Tells this builder to use this map of attributes for the resulting <code>VariantContext</code>
      *
```

Comment:
Document the fact that this method now makes a defensive copy of the map it's passed.